recipes/mender-monitor/debian-master/control

```diff
@@ -13,3 +13,7 @@ Description: Mender Monitor client
  device events to the Mender server, such as log, service files and D-Bus
  signals.
  
+Package: mender-monitor-demo
+Architecture: all
+Depends: mender-monitor, ${misc:Depends}
+Description: Adds demo functionality for the Monitor Client
```

Comment:
Here (and above, in the original package) I would use "Mender Monitor", which is the name of the package and the name we are using in the docs.

Reply:
That description is already present though, keep it out of this PR?

Reply:
You mean the description for `mender-monitor` package? for 1.0.x we should leave it, but for master we should fix it now that we have seen it.